docs/monitoring/spm-faq.md

```diff
@@ -21,11 +21,11 @@ See [integrations](/integration). You can also send [custom metrics](custom-metr
 
 **Can I reduce the amount of logs generated by SPM monitor?**
 
-Yes, starting with 1.20.6 version of SPM client you can make SPM
-monitor reduce the number of messages it logs every minute (otherwise it
-logs detailed info about its monitoring lifecycle). To do that, change
-properties files of your Monitoring Apps. You can find those files
-with:
+Yes, You can make SPM monitor reduce the number of messages it logs every minute 
+(otherwise it logs detailed info about its monitoring lifecycle). To do that, change
+properties files of your Monitoring Apps. 
```

Comment:
properties files => property files
Monitoring Apps => monitoring agents (I assume that is what you meant)